pyodide-build/pyodide_build/pywasmcross.py

```diff
@@ -307,6 +307,7 @@ def replay_genargs_handle_linker_opts(arg: str) -> str | None:
                 "--sysroot=",  # ignore unsupported --sysroot compile argument used in conda
                 "--version-script=",
                 "-R/",  # wasm-ld does not accept -R (runtime libraries)
+                "-R.",  # wasm-ld does not accept -R (runtime libraries)
```

Comment:
LGTM, but the only thing I was wondering about is whether this change I made to add `-R.` to the list of compile options to be removed in `pywasmcross.py` should instead go in this list slightly higher up:
https://github.com/pyodide/pyodide/blob/14708295cd1d2a38235e2dd3727ed7760e29eaa0/pyodide-build/pyodide_build/pywasmcross.py#L294-L303
so only `-R.` exactly gets removed rather than anything that starts with `-R.`. Not sure whether there could be valid options that start with `-R.`. OTOH, maybe leaving it as is is best so things like `-R./another_dir/` also get removed (also not sure whether that could ever happen though).

Reply:
I think it's fine this way since I doubt we would ever want to keep a flag that starts with `-R.`